api/tests/integration/infrastructure/repositories/campaign-management-repository_test.js

```diff
@@ -70,6 +71,41 @@ describe('Integration | Repository | Campaign-Management', () => {
     });
   });
 
+  describe('#update', () => {
+    it('should update the campaign ', async () => {
+      // given
+      const campaign = databaseBuilder.factory.buildCampaign({
+        name: 'Bad campaign',
+        title: null,
+        customLandingPageText: null,
+        customResultPageText: null,
+        customResultPageButtonText: null,
+        customResultPageButtonUrl: null,
+      });
+      await databaseBuilder.commit();
+
+      const campaignAttributes = {
+        name: 'Amazing campaign',
+        title: 'Good title',
+        customLandingPageText: 'End page',
+        customResultPageText: 'Congrats you finished !',
+        customResultPageButtonText: 'Continue here',
+        customResultPageButtonUrl: 'www.next-step.net',
+      };
+      // when
+      await campaignManagementRepository.update({ id: campaign.id, campaignAttributes });
+      const updatedCampaign = await knex('campaigns').where({ id: campaign.id }).first();
+
+      // then
+      expect(updatedCampaign.name).to.equal('Amazing campaign');
+      expect(updatedCampaign.title).to.equal('Good title');
+      expect(updatedCampaign.customLandingPageText).to.equal('End page');
+      expect(updatedCampaign.customResultPageText).to.equal('Congrats you finished !');
+      expect(updatedCampaign.customResultPageButtonText).to.equal('Continue here');
+      expect(updatedCampaign.customResultPageButtonUrl).to.equal('www.next-step.net');
```

Comment:
**suggestion** : vérifier tout en une seule fois `expect(updatedCampaign).to.equal(expectedCampaign)`